dev/run

```diff
@@ -249,9 +249,15 @@ def hack_default_ini(ctx, node, content):
     coffeejs = os.path.join(ctx['rootdir'], "share", "server", "main-coffee.js")
 
     repl = "javascript = %s %s" % (couchjs, mainjs)
+    if os.name == 'nt':
+        # Erlang always wants UNIX-style paths
+        repl = repl.replace("\\", "/")
```

Comment:
This is low level string manipulation which have a corner cases when it wouldn't work.
For example what if user has two slashes or if the path is not absolute.
I don't have access to windows computer to verify but I think `os.path.normcase(path)` should do what you need.